app/Helpers/TopicSupport.php

```diff
@@ -16,7 +16,11 @@
 use App\Events\PromotedDelegatesMailEvent;
 use App\Facades\GetPushNotificationToSupporter;
 use App\Events\NotifyDelegatedAndDelegatorMailEvent;
+<<<<<<< HEAD
+use App\Facades\util;
+=======
 use App\Facades\Util;
+>>>>>>> development
```

Comment:
Remove this lines. It will cause the application to fail

Reply:
@akuks  for this PR i am not getting any conflict on my local on this branch even after merging development to it.